content/docs/atom-guide/atom-sample-viewer/debug-menu-samples.md

```diff
@@ -11,11 +11,54 @@ This section dives into the debug menu samples demonstrated in the Atom Sample V
 {{< /todo >}}
 
 ## CPU Profiler
-The **CPU Profiler**, provides CPU timing information, which includes the time spent working on RHI threads and the time it takes for the GPU to finish its work. The timing is limited to static processes, meaning dynamic processes (such as passes and scopes) will be not be timed nor displayed. The values displayed are instantaneous and represent the timings from the frame prior to the rendering the values. 
 
-To open the CPU Profiler choose `Profile` &rarr; `CPU Profiler`.
+The Atom Renderer's built-in CPU Profiler widget can be used to collect running statistics and visualize profiling data collected from the renderer. 
 
-![CPU Profiler](/images/atom-guide/atom-sample-viewer/cpu-profiler.jpg)
+To open the CPU Profiler widget, click on `Profile` &rarr; `CPU Profiler` from the ImGui debug menu.
+
+### High-level Toolbar
+
+The CPU profiler widget has two views, a statistics view and a visualizer. These two views share a common header for controlling the profiler.
+
+![CPU Profiler Views](/images/atom-guide/atom-sample-viewer/cpu-profiler-views.png)
+
+* Swap to statistics is used to swap between the statistics view and the visualizer.
+* Resume can be used to start or stop profiling data collection from Atom.
+* Capture takes a single-frame capture of CPU profiling data and saves it to a location on disk, by default into /<project>/user/CpuProfiler/.
+* Begin can be used to start a continuous capture of CPU profiling data, during which Atom is storing many frames of data. End will end the continuous capture and save the data that was collected to disk, by default into /<project>/user/CpuProfiler/.
+* Load File will bring up the file picker window which will list all of the.json files in /<project>/user/CpuProfiler/. This list of files is sorted in decreasing order according to modification time, so the most recent capture is at the top.
```

Comment:
Do we want to put these in a table? I also made some slight changes in descriptions. Can we use "profile" instead of "profiling"?

```suggestion
| Tool | Description |
| - | - |
| **Swap to statistics/visualizer** | Swaps between the statistics and visualizer views. |
| **Resume/Pause** | Starts/stops Atom profile data collection. |
| **Capture** | Captures a single frame of CPU profile data. The collected data is saved to `/<project>/user/CpuProfiler/`. |
| **Begin/End** | Starts/stops a capture of continuous frames of CPU profile data. The collected data is saved to `/<project>/user/CpuProfiler/`. |
| **Load file** | Opens the File Picker window, which lists all of the capture files in `/<project>/user/CpuProfiler/`. The files are sorted in descending order according to modification time, with the most recent capture at the top. Refer to [Loading Captures](#loading-captures). |
```